mirna_explorer/repository/create_database.py

```diff
@@ -28,6 +42,35 @@
     print(f'created database in {DB_PATH}')
     db_connection.commit()
 
-db_connection.close()
 
+# populate the seed_match_type table
+db.execute(('INSERT INTO seed_match_types (id, type) '
+            'VALUES(?, ?)'), (0, '8mer'))
+db.execute(('INSERT INTO seed_match_types (id, type) '
+            'VALUES(?, ?)'), (1, '7mer-m8'))
+db.execute(('INSERT INTO seed_match_types (id, type) '
+            'VALUES(?, ?)'), (2, '7mer-A1'))
+
+# populate the mirnas table, here we consider only the human miRNAs
+all_mirnas = pd.read_csv(DATA_FOLDER + MIRNA_FAMILY_INFO_FILE, delimiter='\t')
+# 9606 is the species id for homo sapiens
+human_mirnas = all_mirnas.loc[all_mirnas['Species_ID'] == 9606, :]
+
+with open(DATA_FOLDER + MIRNA_ID_DICTIONARY_FILE) as infile:
+    content = csv.reader(infile, delimiter='\t', quoting=csv.QUOTE_NONE)
+    header = content.__next__()
+    for row in content:
+        mirna_family = row[0]
+        mirna_family = mirna_family
+        mirna_id = int(row[1])
+        sequence = human_mirnas.loc[human_mirnas['MiRBase_ID'] ==
+                                    mirna_family.replace('hsa-mir', 'hsa-miR'),
+                                    'Mature_sequence'].values[0]
+        db.execute(('INSERT INTO mirnas (mirna_id, family, sequence) '
+                    'VALUES(?, ?, ?)'), (mirna_id, mirna_family, sequence))
```

Comment:
In SQLITE we can save database space by not storing IDs. There exists an implicit ID field that autoincrements named `rowid`. i.e. 'FROM mirnas SELECT rowid WHERE filter = hsa-mir-3133`